content/docs/use-cases/data-registries.md

```diff
@@ -21,16 +21,16 @@ Advantages of data registries:
 - **Reusability**: reproduce and organize _feature stores_ with a simple CLI
   (`dvc get` and `dvc import` commands, similar to software package management
   systems like `pip`).
-- **Persistence**: [remote storage](/doc/command-reference/remote) (e.g. an S3
+- **Persistence**: [remote storage](/docs/command-reference/remote) (e.g. an S3
```

Comment:
/doc/ paths are actually correct. /docs/ would just add a redirection step (back to /doc/ )

![image](https://user-images.githubusercontent.com/1477535/133001921-0671c809-4327-45d5-b88b-cbd2b5be48ff.png)

Reply:
From GitHub docs, if I click on the link...it was showing error 404.... maybe it is only working in the website

Reply:
I see. These are source files to render dvc.org. Not meant to be navigated on Github. They just happen to show markdown files as HTML e.g. https://github.com/iterative/dvc.org/blob/master/content/docs/use-cases/data-registries.md but even if you change /doc/ for /docs/ it will give a 404 (would have to be /content/docs/ ).